inference-engine/src/vpu/myriad_plugin/myriad_plugin.cpp

```diff
@@ -37,8 +63,10 @@ ExecutableNetworkInternal::Ptr Engine::LoadExeNetworkImpl(
     parsedConfigCopy.update(config);
 
     auto clonedNetwork = cloneNetwork(network);
+    std::cout << "LoadExeNetworkImpl";
     if (auto function = clonedNetwork->getFunction()) {
-        ngraph::op::GenericIE::DisableReshape noReshape(function);
+        Transformation(clonedNetwork);
```

Comment:
That's incorrect. There should not be conversion to CNN representation before our FrontEnd.

Required points:

1. Common optimization must happen before DTS
1. At the moment of DTS it must be nGraph, not CNN
1. Common optimization do not happen again (in full scope at least) after DTS
1. Conversion to CNN happens exactly at the same moment as before (in FrontEnd before precision conversion)